src/components/articles/preview/ArticlePreview.js

```diff
@@ -0,0 +1,19 @@
+import { Typography, Container } from '@mui/material';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const ArticlePreview = ({article}) => {
+    const navigate = useNavigate();
+
+    const openArticle = () => {
+        navigate(`articles/${article.id}`);
+    }
```

Comment:
ArticlePreview is provided with an `article` which has an ID. This ID is used to open up the article page at `/articles/:id`